runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -354,6 +354,18 @@ public void hotkeyPressed()
 
 			keyManager.registerKeyListener(sidebarListener);
 
+			final HotkeyListener pluginPanelListener = new HotkeyListener(() ->
+				config.panelToggleKey())
```

Comment:
This can be shortened to `final HotkeyListener pluginPanelListener = new HotkeyListener(config::panelToggleKey);`